src/main/java/com/bruberu/gregtechfoodoption/item/GTFOMetaItem.java

```diff
@@ -244,6 +254,13 @@ public void registerSubItems()
 
         UNCOOKED_BACON = addItem(108, "component.bacon");
 
+        GORGONZOLA_WHEEL = addItem(109, "component.gorgonzola_wheel");
+        SALTED_GORGONZOLA_WHEEL = addItem(110, "component.salted_gorgonzola_wheel");
+        SLIGHTLY_AGED_GORGONZOLA_WHEEL = addItem(111, "component.slightly_aged_gorgonzola_wheel");
+        PUNCTURED_GORGONZOLA_WHEEL = addItem(112, "component.punctured_gorgonzola_wheel");
+        FULLY_CURED_GORGONZOLA_WHEEL = addItem(113, "component.fully_cured_gorgonzola_wheel");
+        PENICILINIUM_ROQUEFORTI_CULTURE = addItem(115, "culture.penicilinium");
+        SLICER_BLADE_OCTOGONAL = addItem(116, "config.slicer_blade.octogonal");
```

Comment:
Perhaps you want this to be renamed octagonal, as that is the more common spelling, although octogonal apparently does see some use, although it's mostly archaic.